sdk/nodejs/api/client.gen.ts

```diff
@@ -503,6 +520,8 @@ export class Container extends BaseClient {
 
   /**
    * Initialize this container from this DirectoryID
+   *
+   * @deprecated Replaced by withRootfs.
```

Comment:
@slumbering Is this the correct way to do this in TS?

Reply:
After investigating, yes it seems to be that format.

Reply:
yes it is @aluzzardi